compiler/src/iree/compiler/Codegen/Common/GPU/GPUReduceBankConflicts.cpp

```diff
@@ -45,6 +47,23 @@ static void padAlloc(MLIRContext *context, memref::AllocOp allocOp,
   rewriter.eraseOp(allocOp);
 }
 
+LogicalResult reduceSharedMemoryBankConflicts(mlir::FunctionOpInterface funcOp,
+                                              unsigned paddingSize) {
```

Comment:
I see unsigned in some places and int64_t in some places for paddingSize... doesnt matter, but maybe worth being consistent.